maven-java-11/multimodule-maven-project/userdaomodule/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.baeldung.multimodule-maven-project</groupId>
+        <artifactId>multimodule-maven-project</artifactId>
+        <version>1.0</version>
+    </parent>
+    <groupId>com.baeldung.userdaomodule</groupId>
+    <artifactId>userdaomodule</artifactId>
+    <packaging>jar</packaging>
```

Comment:
Need to add a <version> tag

Reply:
Yes. We need to. Good catch! I'll add it ASAP.